### ISSUE TITLE
False negative when passing a reactive value to a create* or use* function via an object property

### ISSUE THREAD

**Issue by russelldavis** [solidjs-community/eslint-plugin-solid#57]
**Describe the bug**
The `untrackedReactive` error should be generated the same way for an object property as it is for a direct argument, but it isn't.

**To Reproduce**
```js
function createFoo(bar) {}
function createFoo2({bar}) {}

const [bar, setBar] = createSignal();
createFoo(bar()); // error is properly generated
createFoo2({bar: bar()}); // same error should be generated but isn't
```

**Expected behavior**
The same error should be generated in both cases

**Environment (please complete the following information):**
- OS: MacOS 12.6.1
- Node version (`node --version`): v16.18.0
- `eslint-plugin-solid` version (`npm list eslint-plugin-solid`/`yarn why eslint-plugin-solid`): eslint-plugin-solid@0.9.1
- `eslint` version (`npm list eslint`/`yarn why eslint`): eslint@8.30.0

**Additional context**
The issue might have been introduced by this commit: https://github.com/solidjs-community/eslint-plugin-solid/commit/227d02fc8f8f9ca7d62b2d587d5704b03032ad6f

- [ ] I would be willing to contribute a PR to fix this issue

**Comment by ryansolid** [comment]
Verified fixed on current main: `createFoo2({ bar: bar() })` now reports the same `untrackedReactive` warning as the direct-argument form. Thanks for the precise report.
